website_product_document_download_counter/models/product_document.py

```
@@ -0,0 +1,36 @@
+    _inherit = "product.document"
+
+    download_count = fields.Integer(
+        default=0,
+        readonly=True,
+        copy=False,
+        help="Number of times this document has been downloaded",
+    )
+
+    count_downloads_enabled = fields.Boolean(
+        copy=False,
+        help="Enable download counting for this document when published on website",
```

Flake8 (7.2.0)

[error] line too long (84 > 79 characters)

(E501)